docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+
+```
+INFO 087 raft.node: 1 elected leader 1 at term 2 channel=channel1 node=1
+INFO 088 Raft leader changed: 0 -> 1 channel=channel1 node=1
+INFO 089 Start accepting requests as Raft leader at block [0] channel=channel1 node=1
+```
+
+After the channel is started by the first orderer, subsequent nodes can join from either the genesis block or from the latest config block. When an orderer joins from a config block, its status is always "onboarding" while its ledger catches up to the config block that was specified in the join command, after which the status is automatically updated to "active".
+
+Use the `osnadmin channel list` command with the `--channel-id` flag to view the detailed `status` and `consensusRelation` of any **channel** on any ordering node:
+
+```
+osnadmin channel list –-channel-id [CHANNEL_NAME] -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $OSN_TLS_CA_ROOT_CERT --client-cert $ADMIN_TLS_SIGN_CERT --client-key $ADMIN_TLS_PRIVATE_KEY
+```
+
+For example:
+```
+osnadmin channel list –-channel-id channel1 -o HOST2:7081 --ca-file $OSN_TLS_CA_ROOT_CERT --client-cert $ADMIN_TLS_SIGN_CERT --client-key $ADMIN_TLS_PRIVATE_KEY
+```
+
+Replace:
+
+- `CHANNEL_NAME` with the name of the channel specified in the `configtxgen` command when you generated the channel.
+- `ORDERER_ADMIN_LISTENADDRESS` with Orderer.Admin.ListenAddress defined in the orderer.yaml for this orderer.
+- `OSN_TLS_CA_ROOT_CERT` with the path and file name of the orderer organization TLS CA root certificate and intermediate certificate if using an intermediate TLS CA.
+- `ADMIN_TLS_SIGN_CERT` with the path and file name of the admin client signed certificate from the TLS CA.
+- `ADMIN_TLS_PRIVATE_KEY` with the path and file name of the admin client private key from the TLS CA.
+
+The output of this command looks similar to:
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "consensusRelation": "consenter",
+  "status": "active",
+  "height": 1
+}
+```
+
+The following diagram summarizes the steps you have completed:
+
+![create-channel.osnadmin1-3](./osnadmin1-3.png)
+_You used the configtxgen command to create the channel genesis block and provided that file when you ran the osnadmin channel join command for each orderer by targeting the admin endpoint on each node._
+
+Assuming you have successfully run the `osnadmin channel join` on all three ordering nodes, you now have an active channel and the ordering service is ready to order blocks. Peers can join the channel and begin to transact.  
```

Comment:
> ready to order blocks

I would say "is ready to order transactions into blocks"

> Peers can join the channel and begin to transact

I would say "Peers can join the channel and clients can begin to transact"

Reply:
Done.